tensorflow/core/util/gpu_launch_config.h

```diff
@@ -146,7 +147,7 @@ inline CudaLaunchConfig GetCudaLaunchConfig(int work_element_count,
   return GetGpuLaunchConfig(work_element_count, d);
 }
 
-// Calculate the Cuda launch config we should use for a kernel launch. This
+// Calculate the Gpu launch config we should use for a kernel launch. This
```

Comment:
GPU instead of Gpu (admitted, CUDA wasn't cased correctly either).

Reply:
Is the idea to replace every instance of "Gpu" to "GPU" in the comments? I'd assume in the other review #28568, update "Cuda" to "CUDA" in all comments as well. Just double check to make sure.

BTW it make me curious whether this is this part of a rule of a coding convention?

Reply:
Updated all instance of "Gpu" to "GPU" in the comments, let me know if this is expected. Thanks

Reply:
Correct, GPU and CUDA in comments, Gpu and Cuda in code. From https://google.github.io/styleguide/cppguide.html:

> When abbreviations or acronyms appear in such names, prefer to capitalize the abbreviations or acronyms as single words (i.e StartRpc(), not StartRPC()).

Reply:
Thanks for the reply! I'll need get myself a bit more familiar with the cpp style guide.